keps/sig-node/1769-memory-manager/README.md

```diff
@@ -435,21 +501,19 @@ Memory pinning will be validated for Topology Manager `single-numa-node` and `re
 
 #### GA (stable)
 
-- User feedback
-- TBD
-
-<!--
-#### Removing a deprecated flag
+- N examples of real-world usage
+- N installs
+- More rigorous forms of testing—e.g., downgrade tests and scalability tests
+- Allowing time for feedback
 
-- Announce deprecation and support policy of the existing flag
-- Two versions passed since introducing the functionality which deprecates the flag (to address version skew)
-- Address feedback on usage/changed behavior, provided on GitHub issues
-- Deprecate the flag
+**Note:** Generally we also wait at least two releases between beta and
```

Comment:
we do not need this note since we are in beta for longer than this

Reply:
removed, thanks